drivers/ukintctlr/gic/Config.uk

```diff
@@ -4,10 +4,10 @@ config LIBUKINTCTLR_GIC
 
 config LIBUKINTCTLR_GICV2
 	bool "Arm Generic Interrupt Controller (GICv2)"
-	depends on ARCH_ARM_64
+	depends on HAVE_INTCTLR
```

Comment:
This change allows selecting the GIC on x86, which I think is not very useful.

Reply:
Yes, I noticed too. I'll fix that thanks 👍🏼